resolwe_bio/docker_images/legacy/packages-manual/primerclip.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+. /var/cache/build/packages-manual/common.sh
+
+download_and_verify \
+    genialis \
+    primerclip \
+    2 \
+    5b74d96bab9bff0b7880ca134c4d67c67230e7d36f161269282d5994254da3c4 \
+    'https://data.genialis.com/repo/3f3f14d1-5100-46d5-af7b-abca7ff4a48f/292c647949ed795d6c73d8088cb839586e2287c6/?file_name=primerclip-v${version}&op=download&t=ca7225aa30&p=/tools/primerclip-v${version}#.elf'
```

Comment:
Replace the links to data.genialis.com with links to data2.genialis.com. Ask @abukosek where are the files on the new server and what are the links to them.